Mathlib/NumberTheory/NumberField/FractionalIdeal.lean

```diff
@@ -0,0 +1,112 @@
+/-
+Copyright (c) 2024 Xavier Roblot. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Xavier Roblot
+-/
+import Mathlib.NumberTheory.NumberField.Basic
+import Mathlib.RingTheory.FractionalIdeal.Norm
+
+/-!
+
+# Fractional ideals of number fields
+
+Prove some results on the fractional ideals of number fields.
+
+## Main definitions and results
+
+  * `NumberField.basisOfFractionalIdeal`: A `ℚ`-basis of `K` that spans `I` over `ℤ` where `I` is
+  a fractional ideal of a number field `K`.
+  * `NumberField.det_basisOfFractionalIdeal_eq_absNorm`: for `I` a fractional ideal of a number
```

Comment:
Can you modify a little bit the docstring using standard math terminology? I mean, this is the fact that the determinant of the base change matrix is the norm. It's also a good idea to add it right before the statement.